models/public/pspnet-pytorch/model.yml

```diff
@@ -0,0 +1,160 @@
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: models/mmcv-1.2.0.tar.gz
+  - $type: unpack_archive
+    format: zip
+    file: models/addict-2.4.0-py3-none-any.whl
+  - $type: unpack_archive
+    format: zip
+    file: models/mmsegmentation-0.11.0-py3-none-any.whl
+
+  - $type: regex_replace
+    pattern: '''../_base_/schedules/schedule_20k.py'''
+    replacement: '# \g<0>'
+    file: models/configs/pspnet/pspnet_r50-d8_512x512_20k_voc12aug.py
+  - $type: regex_replace
+    pattern: 'isinstance\(var, list\)'
+    replacement: 'isinstance(var, (list, torch.Tensor))'
+    file: models/mmseg/models/segmentors/base.py
+  - $type: regex_replace
+    pattern: 'num_augs = len\(imgs\)'
+    replacement: 'num_augs = 1'
+    file: models/mmseg/models/segmentors/base.py
+  - $type: regex_replace
+    pattern: 'imgs\[0\]'
+    replacement: 'imgs'
+    file: models/mmseg/models/segmentors/base.py
+  - $type: regex_replace
+    pattern: 'from (?!.resnet)'
+    replacement: '# \g<0>'
+    file: models/mmseg/models/backbones/__init__.py
+  - $type: regex_replace
+    pattern: '''(?!ResNet)\w+'',?'
+    replacement: ''
+    file: models/mmseg/models/backbones/__init__.py
+  - $type: regex_replace
+    pattern: 'from (?!(.fcn_head|.psp_head))'
+    replacement: '# \g<0>'
+    file: models/mmseg/models/decode_heads/__init__.py
+  - $type: regex_replace
+    pattern: '''(?!FCN|PSP)\w+'',?'
+    replacement: ''
+    file: models/mmseg/models/decode_heads/__init__.py
+  - $type: regex_replace
+    pattern: 'from yapf'
+    replacement: '# \g<0>'
+    file: models/mmcv-1.2.0/mmcv/utils/config.py
+  - $type: regex_replace
+    pattern: 'from (.video|.visualization|.fileio|.image)'
+    replacement: '# \g<0>'
+    file: models/mmcv-1.2.0/mmcv/__init__.py
+  - $type: regex_replace
+    pattern: 'import cv2'
+    replacement: '# \g<0>'
+    file: models/mmcv-1.2.0/mmcv/utils/env.py
+  - $type: regex_replace
+    pattern: 'import (regex as re|re)'
+    replacement: 'pass'
+    file: models/mmcv-1.2.0/mmcv/cnn/bricks/plugin.py
+  - $type: regex_replace
+    pattern: '(import (regex as re|re)|shutil.copyfile.+)'
+    replacement: 'pass'
+    file: models/mmcv-1.2.0/mmcv/utils/config.py
+  - $type: regex_replace
+    pattern: '(.+re.sub.+|.+osp.basename\(temp_config_file.name\))'
+    replacement: '# \g<0>'
+    file: models/mmcv-1.2.0/mmcv/utils/config.py
+  - $type: regex_replace
+    pattern: 'use_predefined_variables=True'
+    replacement: 'use_predefined_variables=False'
+    file: models/mmcv-1.2.0/mmcv/utils/config.py
+  - $type: regex_replace
+    pattern: 'osp.splitext\(temp_config_name\)\[0\]'
+    replacement: 'osp.splitext(osp.basename(filename))[0]'
+    file: models/mmcv-1.2.0/mmcv/utils/config.py
+  - $type: regex_replace
+    pattern: 'sys.path.insert\(0, temp_config_dir\)'
+    replacement: 'sys.path.insert(0, osp.dirname(filename))'
+    file: models/mmcv-1.2.0/mmcv/utils/config.py
```

Comment:
I wonder if all this is really necessary. What if you just replace `import regex as re` with `import re` and that's it? I examined the usage of `regex` in mmcv, and it looks like it was initially introduced to resolve open-mmlab/mmaction2#49, but then open-mmlab/mmcv#513 was done, which seems to resolve the same issue, so maybe `regex` isn't needed anymore.

If you do remove this, make sure you test it on Windows, since that's where the original code used `regex`.

Reply:
Done. Tested on Windows - works